config/initializers/comfy_patching.rb

```diff
@@ -29,7 +29,19 @@ def assign_layout_categories
         end
       end
 
-      delete_orphan_categories && return if _categories.blank?
+      # Two situations for deleting orphan categories, either _categories has no or a restricted set of the currently existing categories
+      # Or, _categories consists of tags with entirely new labels, and hence new layouts would have to be created
+      # Not handled at present - adding new categories which equates to adding new layout categories which means _layout_category is empty
+      # _categories = [{ tag_params: 'topics', tag_class: 'categories'}]
+      # layouts_categories = [{label: 'topics'...}, {label: 'types'...}]
+      # orphan_categories = [{label: 'types'}]
+      layouts_categories = Comfy::Cms::LayoutsCategory.where(layout_id: self.id)
+      .map(&:layout_category)
```

Comment:
I would add this in the same line above or add some indentation here (2 spaces) so to understand that it's something referring to a preceding code